packages/datadog-instrumentations/src/helpers/instrument.js

```diff
@@ -21,11 +21,17 @@ exports.channel = function (name) {
  * @param Function hook
  */
 exports.addHook = function addHook ({ name, versions, file }, hook) {
-  if (!instrumentations[name]) {
-    instrumentations[name] = []
+  if (typeof name === 'string') {
+    name = [name]
   }
 
-  instrumentations[name].push({ name, versions, file, hook })
+  name.forEach(val => {
+    if (!instrumentations[val]) {
+      instrumentations[val] = []
+    }
+
+    instrumentations[val].push({ name: val, versions, file, hook })
+  })
```

Comment:
I would generally recommend for-of over forEach.

```suggestion
  for (const val of name) {
    if (!instrumentations[val]) {
      instrumentations[val] = []
    }

    instrumentations[val].push({ name: val, versions, file, hook })
  }
```